src/test/java/com/ibm/watson/developer_cloud/text_to_speech/v1/CustomizationsIT.java

```diff
@@ -0,0 +1,212 @@
+package com.ibm.watson.developer_cloud.text_to_speech.v1;
+
+import static com.ibm.watson.developer_cloud.text_to_speech.v1.model.AudioFormat.WAV;
+import static com.ibm.watson.developer_cloud.text_to_speech.v1.model.Voice.EN_MICHAEL;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+import org.apache.commons.io.IOUtils;
```

Comment:
It's a transitive dependency (`org.mock-server:mockserver-core` > `commons-io`). Since we're stopping to use org.mock-server, I implemented the method on my own.